repositories/fileMovieRepository.js

```diff
@@ -75,8 +75,19 @@ class FileMovieRepository {
 
   async addAllMovies(newMovies) {
     const movies = await this._fetch_movies();
+    
+    // Find the current max ID in the movies array to avoid conflicts
+    let maxId = movies.length > 0 ? Math.max(...movies.map(movie => movie.id)) : 0;
```

Comment:
Can't we use the id of the last movie as the max id?

Reply:
Yes, but what if there are no movies then there won't be an ID. so the movie length - 1 you suggested might not work.